services/backend/src/service/connectivity.js

```diff
@@ -42,13 +44,46 @@ module.exports = () => {
      */
     getPAConnectivityLayers: async () => connectivityPALayers,
 
+    /**
+     * Get the values through time of a protected area connectivity category in a given area
+     *
+     * @param {String | Number} areaType area type
+     * @param {String} areaId area id
+     * @param {String} category category
```

Comment:
Please add a better descripcion for the parameter "category"

Reply:
It was replaced with category of the connectivity index